ts/nni_manager/common/trial_keeper/keeper.ts

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ *  The helper class to implement a "reuse" training service.
+ *
+ *  TrialKeeper has a very similar interface to TrainingSerivceV3.
+ *  In fact the local training service is a thin wrapper over TrialKeeper.
+ **/
+
+import { EventEmitter } from 'events';
+import fs from 'fs/promises';
+import path from 'path';
+
+import type { Command } from 'common/command_channel/interface';
+import { HttpChannelServer } from 'common/command_channel/http';
+import globals from 'common/globals';
+import { Logger, getLogger } from 'common/log';
+import { TrialProcess, TrialProcessOptions } from './process';
+import { GpuSchedulerClient } from './gpu_scheduler_client';
+
+export declare namespace TrialKeeper {
+    export interface TrialOptions {
+        id: string;
+        command: string;
```

Comment:
any idea about distributed trial? what does `command` look like for distributed trial?